Triangulation_2/include/CGAL/Triangulation_2.h

```diff
@@ -243,6 +243,7 @@ class Triangulation_2
                   const Geom_traits& geom_traits=Geom_traits())
     : _gt(geom_traits)
   {
+    _infinite_vertex = _tds.insert_first();
     insert(first,last);
   }
```

Comment:
Since the branch is for CGAL-5.0, with C++11, here is a very good example for the introduction of [delegating constructors]:
```C++
  template <class InputIterator>
  Triangulation_2(InputIterator first, InputIterator last,
                  const Geom_traits& geom_traits=Geom_traits())
    : Triangulation_2(geom_traits)
  {
    insert(first,last);
  }
```

[delegating constructors]: https://en.cppreference.com/w/cpp/language/initializer_list#Delegating_constructor

That way, without knowing about the internals, that constructor calls the constructor with one argument, and then call `insert(first, last)`.